src/service/config.py

```diff
@@ -0,0 +1,52 @@
+"""
+Configuration settings for the Spark Manager API.
+"""
+
+import logging
+import os
+from functools import lru_cache
+
+from pydantic import Field
+from pydantic_settings import BaseSettings
+
+APP_VERSION = "0.0.1"
```

Comment:
Semantic versioning generally starts at 0.1.0 https://semver.org/#how-should-i-deal-with-revisions-in-the-0yz-initial-development-phase

Reply:
👍